src/utils/theme_utils.js

```diff
@@ -16,9 +16,24 @@ export function makeStyleFromTheme(getStyleFromTheme: (Object) => Object): (Obje
     };
 }
 
-function normalizeColor(oldColor: string): string {
-    let color = oldColor;
-    if (color.length && color[0] === '#') {
+const rgbPattern = /^rgba?\(([\d.]+),([\d.]+),([\d.]+)(?:,([\d.]+))?\)$/;
```

Comment:
Good point. I'll take out the decimals from those since we're not parsing them as floats. I think React Native may support floats there, but I'll use ints to keep the behaviour the same between web app and mobile